server/node.go

```diff
@@ -92,6 +92,14 @@ func allocateStoreIDs(nodeID roachpb.NodeID, inc int64, db *client.DB) (roachpb.
 	return roachpb.StoreID(r.ValueInt() - inc + 1), nil
 }
 
+// GetBootstrapSchema returns the schema which will be used to bootstrap a new
+// server.
+func GetBootstrapSchema() sql.MetadataSchema {
```

Comment:
This separation seems unnecessary; I'd rather just put all predefined tables in one place.